app/core/voyagerUtils.js

```diff
@@ -0,0 +1,93 @@
+
+module.exports = {
+
+  /**
+   * Creates a level status map using the list of level session objects.
+   * @param {Object[]} sessions - The list of level session objects.
+   * @returns {Object} - Object with key as the level original id, and value as complete/started.
+   */
+  getLevelStatusMap: function (sessions) {
+    let levelStatusMap = {}
```

Comment:
As this isn't being rebound, should we prefer `const`?
```suggestion
    const levelStatusMap = {}
```